package.json

```diff
@@ -40,7 +40,8 @@
     "fix:lockfile": "pnpm install --fix-lockfile",
     "update:lockfile": "pnpm install --lockfile-only",
     "check-changeset": "cd ./scripts/check-changeset && pnpm start",
-    "update-codesmith": "cd ./scripts/update-codesmith && pnpm start"
+    "update-codesmith": "cd ./scripts/update-codesmith && pnpm start",
+    "update-turbo-cache": "turbo run build --force --filter {packages/**}"
```

Comment:
It may be used if a build produces content that affects the correctness of the product. For example, a situation where files and folders with the same name exist in the product because of a change.

Reply:
get~